2.0/collections.md

```diff
@@ -2,6 +2,24 @@
 
 Collections attach methods to arrays and dictionaries so that you can manipulate them with ease. Under the hood `Lucid` also returns a copy of collection when you fetch values from your database.
 
+## Registering
```

Comment:
Rename it to setup to keep it consistent with other docs. Also add the setup link inside the TOC.